src/Directory.Packages.props

```diff
@@ -22,10 +20,10 @@
     <PackageVersion Include="NUnit.Analyzers" Version="4.2.0" />
     <PackageVersion Include="NUnit3TestAdapter" Version="4.5.0" />
     <PackageVersion Include="Particular.Approvals" Version="1.0.0" />
+    <PackageVersion Include="Particular.LicensingComponent.Report" Version="1.0.0-alpha.2" />
```

Comment:
it seems the package is still alpha, are planning to release the package first?

Reply:
yep did that this morning